capsules/src/ble.rs

```diff
@@ -172,10 +167,10 @@ impl<'a, R: BleAdvertisementDriver + 'a, A: hil::time::Alarm + 'a> BLE<'a, R, A>
         ReturnCode::SUCCESS
     }
 
-    pub fn configure_periodic_alarm(&self) {
+    fn configure_periodic_alarm(&self) {
         self.radio.set_channel(37);
-        let tics = self.alarm.now().wrapping_add(5017 as u32);
-        self.alarm.set_alarm(tics);
+        let ms_in_tics = self.alarm.now().wrapping_add(self.interval.get());
```

Comment:
this was probably a bad name, tested in a hurry and forgot to change to something better